channel/src/test/java/com/synopsys/integration/alert/channel/email/action/EmailGlobalTestActionTest.java

```diff
@@ -146,27 +134,23 @@ private EmailChannelMessageSender createValidEmailChannelMessageSender(String em
+        return new EmailChannelMessageSender(ChannelKeys.EMAIL, testAlertProperties, emailAddressGatherer, emailAttachmentFileCreator, freemarkerTemplatingService, null);
     }
 
-    private ConfigurationModel createValidEmailGlobalConfigModel(TestProperties testProperties) {
+    private FieldUtility createValidEmailGlobalFieldUtility(TestProperties testProperties) {
         Map<String, ConfigurationFieldModel> configuredFields = new HashMap<>();
         addConfigurationFieldToMap(configuredFields, EmailPropertyKeys.JAVAMAIL_HOST_KEY.getPropertyKey(), testProperties.getProperty(TestPropertyKey.TEST_EMAIL_SMTP_HOST));
         addConfigurationFieldToMap(configuredFields, EmailPropertyKeys.JAVAMAIL_FROM_KEY.getPropertyKey(), testProperties.getProperty(TestPropertyKey.TEST_EMAIL_SMTP_FROM));
-        addConfigurationFieldToMap(configuredFields, EmailPropertyKeys.JAVAMAIL_USER_KEY.getPropertyKey(), testProperties.getProperty(TestPropertyKey.TEST_EMAIL_SMTP_USER));
-        addConfigurationFieldToMap(configuredFields, EmailPropertyKeys.JAVAMAIL_PASSWORD_KEY.getPropertyKey(), testProperties.getProperty(TestPropertyKey.TEST_EMAIL_SMTP_PASSWORD));
-        addConfigurationFieldToMap(configuredFields, EmailPropertyKeys.JAVAMAIL_EHLO_KEY.getPropertyKey(), testProperties.getProperty(TestPropertyKey.TEST_EMAIL_SMTP_EHLO));
-        addConfigurationFieldToMap(configuredFields, EmailPropertyKeys.JAVAMAIL_AUTH_KEY.getPropertyKey(), testProperties.getProperty(TestPropertyKey.TEST_EMAIL_SMTP_AUTH));
-        addConfigurationFieldToMap(configuredFields, EmailPropertyKeys.JAVAMAIL_PORT_KEY.getPropertyKey(), testProperties.getProperty(TestPropertyKey.TEST_EMAIL_SMTP_PORT));
 
-        return new ConfigurationModel(0L, 0L, "", "", ConfigContextEnum.GLOBAL, configuredFields);
+        testProperties.getOptionalProperty(TestPropertyKey.TEST_EMAIL_SMTP_USER).ifPresent(prop -> addConfigurationFieldToMap(configuredFields, EmailPropertyKeys.JAVAMAIL_USER_KEY.getPropertyKey(), prop));
+        testProperties.getOptionalProperty(TestPropertyKey.TEST_EMAIL_SMTP_PASSWORD).ifPresent(prop -> addConfigurationFieldToMap(configuredFields, EmailPropertyKeys.JAVAMAIL_PASSWORD_KEY.getPropertyKey(), prop));
+        testProperties.getOptionalProperty(TestPropertyKey.TEST_EMAIL_SMTP_EHLO).ifPresent(prop -> addConfigurationFieldToMap(configuredFields, EmailPropertyKeys.JAVAMAIL_EHLO_KEY.getPropertyKey(), prop));
+        testProperties.getOptionalProperty(TestPropertyKey.TEST_EMAIL_SMTP_AUTH).ifPresent(prop -> addConfigurationFieldToMap(configuredFields, EmailPropertyKeys.JAVAMAIL_AUTH_KEY.getPropertyKey(), prop));
+        testProperties.getOptionalProperty(TestPropertyKey.TEST_EMAIL_SMTP_PORT).ifPresent(prop -> addConfigurationFieldToMap(configuredFields, EmailPropertyKeys.JAVAMAIL_PORT_KEY.getPropertyKey(), prop));
```

Comment:
Previously, you had to provide all of these properties even if you didn't need them.